src/sections/contact/Contact.styled.tsx

```diff
@@ -45,9 +43,19 @@ export const ContactHalf = styled.span`
 
 export const ContactItem = styled.div``;
 
-export const ContactAction = styled.a`
-  color: ${theme.colors.dark};
-  @media screen and (${theme.breakpoints.gteDesktopLgQuery}) {
-    color: ${theme.colors.ice};
-  }
-`;
+export const ContactAction = styled.a<{
+  tickets?: boolean;
+}>(
+  ({ tickets }) => css`
+    ${tickets &&
+    css`
+      color: white !important;
```

Comment:
is there a need for `!important` ?

Reply:
Yes, but it is true, I can also switch condition with color prop below